api/content.go

```diff
@@ -11,55 +12,103 @@ import (
 	"github.com/gorilla/mux"
 )
 
-func addItem(contentList *models.ContentResponseAPI, typeName string, itemLink *[]models.TypeLinkObject, id string, state string, privateResponse bool) {
-	if itemLink == nil {
-		return
+// Flag values for a query type:
+const (
```

Comment:
Consider moving the consts all the way through getContentTypeParameter to a separate file specifically for query parameter logic? It feels like its own little chunk and would allow easier access straight to the handlers

Reply:
done.